example/App.tsx

```diff
@@ -1,42 +1,167 @@
-import React from "react"
-import { View } from "react-native"
+import React, { useState, FC } from "react"
+import { SafeAreaView, Text, View, StyleSheet, StyleProp, ViewStyle } from "react-native"
+import { NativeRouter, Routes, Route, Link } from "react-router-native"
+
+import {
+  BudgetsWidget,
+  ConnectWidget,
+  GoalsWidget,
+  PulseWidget,
+  SpendingWidget,
+  TransactionsWidget,
+} from "@mxenabled/react-native-widget-sdk"
 
-import { ConnectWidget } from "@mxenabled/react-native-widget-sdk"
 import config from "./config.json"
 
+const styles = StyleSheet.create({
+  page: {
+    backgroundColor: "#ffffff",
+    paddingTop: 10,
+  },
+  heading: {
+    paddingLeft: 25,
+  },
+  navigation: {
+    flex: 1,
+    flexWrap: "wrap",
+    padding: 10,
+  },
+  navigationButton: {
+    alignItems: "center",
+    backgroundColor: "#fbfbfb",
+    margin: 10,
+    padding: 20,
+  },
+  navItemNormalText: {
+    color: "#2980b9",
+    fontSize: 20,
+  },
+  back: {
+    color: "#2980b9",
+    fontSize: 15,
+    marginTop: 10,
+    marginBottom: 10,
+  },
+})
+
 export default function App() {
   return (
-    <View style={{ flex: 1 }}>
+    <NativeRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/connect" element={<Connect />} />
+        <Route path="/budgets" element={<Budgets />} />
+        <Route path="/goals" element={<Goals />} />
+        <Route path="/pulse" element={<Pulse />} />
+        <Route path="/spending" element={<Spending />} />
+        <Route path="/transactions" element={<Transactions />} />
+      </Routes>
+    </NativeRouter>
+  )
+}
+
+const Home = () => (
+  <Page style={styles.navigation} goBack={false}>
+    <NavigationButton to="/connect">Connect</NavigationButton>
```

Comment:
Would it be simpler to use the RN flatlist for this? https://reactnative.dev/docs/flatlist

Reply:
@inphomercial looks interesting, how are you thinking it'd be simpler though?